### ISSUE TITLE
Issue with latest version

### ISSUE THREAD

**Issue by LeCodeCo** [appleboy/lambda-action#14]
The new update seems to have broken my actions.

```
Download action repository 'appleboy/lambda-action@master'
Error: appleboy/lambda-action/master/action.yml:
Error: appleboy/lambda-action/master/action.yml: (Line: 38, Col: 98, Idx: 1411) - (Line: 38, Col: 210, Idx: 1523): While parsing a block mapping, did not find expected key.
Error: System.ArgumentException: Unexpected type '' encountered while reading 'action manifest root'. The type 'MappingToken' was expected.
   at GitHub.DistributedTask.ObjectTemplating.Tokens.TemplateTokenExtensions.AssertMapping(TemplateToken value, String objectDescription)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Fail to load appleboy/lambda-action/master/action.yml
```

**Comment by aleon68** [comment]
I have problem with new version

Run appleboy/lambda-action@master
2020/10/24 03:39:22 AccessDeniedException: 
	status code: 403, request id: 88525d89-5162-461c-9df0-751443ef394b
2020/10/24 03:39:22 AccessDeniedException: 
	status code: 403, request id: 88525d89-5162-461c-9df0-751443ef394b

I don't change anything, check user and credentials, but don't work

**Comment by appleboy** [comment]
fixed master branch and create new release [v0.0.6](https://github.com/appleboy/lambda-action/releases/tag/v0.0.6)
